CriteoPublisherSdk/Tests/IntegrationTests/CR_BidManagerIntegrationTests.m

```diff
@@ -147,6 +147,14 @@ - (void)test_givenPrefetchingBid_whenExpiredBid_ShouldBeRemovedFromCache {
   [self checkAnotherPrefetchPopulateCache];
 }
 
+- (void)test_givenPrefetchingBid_whenSilenceModeBid_ShouldNotPopulateCache {
+  [self givenMockedCdbResponseBids:@[ self.silenceModeBid ]];
+  [self whenPrefetchingBid];
+  [self shouldNotPopulateCache];
+
+  [self checkAnotherPrefetchPopulateCache];
```

Comment:
This is true only when the silence TTL is over. In this case, TTL=10ms, and I'm afraid that this might be flaky.
Do you mind adding a sleep or a stub time to make this test deterministic ?